### PR TITLE
Modified physics/drag_suite.F90 to remove topographic limitations on small-scale GWD and TOFD

### PR DESCRIPTION
**Pull request by mdtoyNOAA**
This is an enhancement to two GSL drag suite components:  1) Small-scale gravity wave drag, and 2) Turbulent orographic form drag (TOFD).  The limits to orographic variance was eliminated and the e-folding height of the TOFD scheme is returned to it's original height of 1.5km.  The results improved 10m windspeed verification in a May 2021 retro test as shown in the attached report (*.pdf).  The enhancements are shown by "Experiment 3" in the report.

Regression tests were performed to generate baseline files.  The tests passed on Hera (Intel and GNU compilers) and on Jet (Intel compiler).  The results are in the corresponding tests/RegressionsTests*log files.

The baseline files are on Hera at:
/scratch1/BMC/wrfruc/mtoy/stmp4/Michael.Toy/FV3_RT/REGRESSION_TEST_INTEL/
/scratch1/BMC/wrfruc/mtoy/stmp4/Michael.Toy/FV3_RT/REGRESSION_TEST_GNU/

And on Jet at:
/lfs4/BMC/wrfruc/mtoy/RT_BASELINE/Michael.Toy/FV3_RT/REGRESSION_TEST_INTEL/

[RRFS_GWD_retro_results_2021_12_03.pdf](https://github.com/NOAA-GSL/ccpp-physics/files/8219444/RRFS_GWD_retro_results_2021_12_03.pdf)

### CODE REVIEW

**Review by SamuelTrahanNOAA — APPROVED** [review]
These changes should use namelist flags instead of commented-out code. If this were any other branch, I would certainly reject the changes. You'll need to clean this up before it goes to gsl/develop.

**Comment by mdtoyNOAA** [comment]
Sam,

I understand what you mean.  I can delete the commented-out code, as there’s no need for a namelist option switch between the old and new versions.  When would be the best time to do this?  Thanks.

— Mike

**Comment by SamuelTrahanNOAA** [comment]
> I understand what you mean.  I can delete the commented-out code, as there’s no need for a namelist option switch between the old and new versions.  When would be the best time to do this?  Thanks.

Is that code used in other suites? If so, it has to remain in and enabled by default.

**Comment by mdtoyNOAA** [comment]
No, the commented out code will not be used by other suites.

**Comment by SamuelTrahanNOAA** [comment]
We should combine that clean-up with my PR that will update the CODEOWNERS. Could you please make a branch with those changes *after* @tanyasmirnova's changes are merged to RRFS_dev? Then I'll copy your branch and put the new CODEOWNERS in it, and do a PR.

**Comment by mdtoyNOAA** [comment]
Hi Sam,

I’m checking in to see if Tanya’s changes have been merged into RRFS_dev.  I can then clean up my code and merge with her branch and issue a PR.  Let me know.  I’ll be on leave next week, but if need be, I can do the code update then.

Thanks,
Mike




> On Mar 10, 2022, at 9:48 AM, Samuel Trahan (NOAA contractor) ***@***.***> wrote:
> 
> 
> We should combine that clean-up with my PR that will update the CODEOWNERS. Could you please make a branch with those changes after @tanyasmirnova <https://github.com/tanyasmirnova>'s changes are merged to RRFS_dev? Then I'll copy your branch and put the new CODEOWNERS in it, and do a PR.
> 



@SamuelTrahanNOAA approved this pull request.

These changes should use namelist flags instead of commented-out code. If this were any other branch, I would certainly reject the changes. You'll need to clean this up before it goes to gsl/develop.

**Pull request merged**
